command/analyzer/dryrun/source_test.go

```diff
@@ -72,7 +72,7 @@ func TestPrepareAnalysisConfig(t *testing.T) {
 			t.Errorf("Failed to read the expected analysis config. Error:%s", err)
 		}
 
-		if !bytes.Equal(receivedAnalysisConfig, expectedAnalysisConfig) {
+		if cmp.Equal(receivedAnalysisConfig, expectedAnalysisConfig) {
```

Comment:
consider doing `!= 0` so the intent is clear.

```suggestion
		if cmp.Equal(receivedAnalysisConfig, expectedAnalysisConfig) != 0 {
```

Reply:
Doesn't `cmp.Equal` already return a `bool`? 🤔

Reply:
Nice catch. Thanks! `s/cmp.Equal/!cmp.Equal`